maintainers/maintainer-list.nix

```diff
@@ -8767,6 +8767,12 @@
     githubId = 1181362;
     name = "Stefan Junker";
   };
+  stevenroose = {
+    email = "github@stevenroose.org";
+    github = "stevenroose";
+    githubId = 853468;
+    name = "Steven Roose";
+  };
```

Comment:
Can you split this into a seperate commit?

Reply:
Hmm is that common? I can do that, yes.

Reply:
Awesome, thanks a lot.
It just feels better having this seperated from the other changes. Feels more clean! <3